packages/nimble-components/src/rich-text-editor/index.ts

```diff
@@ -294,6 +402,8 @@ export class RichTextEditor extends FoundationElement {
         this.editor.className = 'editor';
         this.editor.setAttribute('aria-multiline', 'true');
         this.editor.setAttribute('role', 'textbox');
+        this.editor.setAttribute('aria-disabled', 'false');
+        this.editor.setAttribute('aria-label', 'Rich Text Editor');
```

Comment:
We can't just set `aria-label` to an unlocalized constant string. We should [apply the `ARIAGlobalStatesAndProperties` mixin](https://github.com/ni/nimble/blob/12beb26dae4b4151fa3d06a1e6f0ec8fabd52b19/packages/nimble-components/src/drawer/index.ts#L136) and then [reflect the `ariaLabel` property value to the inner editor element's `aria-label` attribute](https://github.com/ni/nimble/blob/12beb26dae4b4151fa3d06a1e6f0ec8fabd52b19/packages/nimble-components/src/drawer/template.ts#L7).

Reply:
Thanks for the references, I have updated the code by applying the mixin and reflecting the `aria-label` value to the inner element. As we are not directly binding the value to the template, I used `ariaLabelChanged` to update the attribute value.